tests/CatalogTests/CatalogTests.csproj

```diff
@@ -175,6 +175,12 @@
     <None Include="TestData\OneValidDependencyOneEmptyIdWithGroups.0.1.0.nupkg">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="TestData\TestPackage.LicenseExpression.0.1.0.nupkg">
+	 <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
>	 [](start = 0, length = 1)

Please use spaces instead of tabs. Same comment for line 182

Reply:
Done.